app/components/agenda/agendaitem/agendaitem-decision.js

```diff
@@ -8,6 +8,7 @@ import CONSTANTS from 'frontend-kaleidos/config/constants';
 import addLeadingZeros from 'frontend-kaleidos/utils/add-leading-zeros';
 import VRDocumentName from 'frontend-kaleidos/utils/vr-document-name';
 import { deleteFile } from 'frontend-kaleidos/utils/document-delete-helpers';
+import { generateDecisionReport } from 'frontend-kaleidos/utils/generate-pdf';
```

Comment:
Noticed the new https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/development/app/services/decision-report-generation.js, will replace this with that ASAP, and probably add one for minutes generation/adapt the existing service to also generate the minutes.

Reply:
https://github.com/kanselarij-vlaanderen/frontend-kaleidos/pull/1887 I implemented the service for the minutes generation here